pwa/src/templates/templateParts/cronjobsForm/EditCronjobFormTemplate.tsx

```diff
@@ -70,6 +73,9 @@ export const EditCronjobFormTemplate: React.FC<EditCronjobFormTemplateProps> = (
       "throws",
       cronjob["throws"]?.map((_throw: any) => ({ label: _throw, value: _throw })),
     );
+
+    setNextRun(cronjob.nextRun);
+    setLastRun(cronjob.lastRun);
```

Comment:
I think we could skip this explicit `set...` and just set them when we initialise them, like so:

`const [nextRun, setNextRun] = React.useState<string>(cronjob.lastRun);`

Which in turn could become: `const [nextRun] = React.useState<string>(cronjob.lastRun);`

Reply:
And then, we could maybe skip the `state` altogether, like so:

`<Tag label={dateTimeTag(t(i18n.language), cronjob.nextRun) ?? "-"} />`